docs/pages/setup/admin/daemon.mdx

```diff
@@ -33,89 +36,106 @@ $ readlink /sbin/init
+Active: active (running) since Mon 2022-04-18 18:33:41 UTC; 41s ago
 ```
 
-You can confirm that Teleport is running as a systemd service with the following
-command, which should show an `Active` status of `active (running)`:
+### Creating the systemd Daemon When Installed From Source
+
+Systemd is designed to use **unit files** which are the primary file used to define and set the conditions for
+any needed service daemon on systemd, in this case the `teleport` service. In order to more easily support systemd configurations, Teleport is capable of recommending and assisting with the installation of a default
+unit file using as little as one command (`teleport install systemd -o /unit/file/load/path`).
+
+Enter the following teleport command to have teleport output a systemd unit file (using `stdout`) to the terminal to be reviewed and applied as needed:
 
 ```code
-$ sudo systemctl status teleport.service
-● teleport.service - Teleport SSH Service
-     Loaded: loaded (/lib/systemd/system/teleport.service; enabled; vendor preset: enabled)
-     Active: active (running) since Mon 2022-04-18 18:33:41 UTC; 41s ago
-   Main PID: 442 (teleport)
-      Tasks: 9 (limit: 1116)
-     Memory: 116.9M
-     CGroup: /system.slice/teleport.service
-             └─442 /usr/local/bin/teleport start --pid-file=/run/teleport.pid
-
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:SER] SSH proxy service 9.0.4:v9.0.4-0-gf577413>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:SER] SSH proxy service 9.0.4:v9.0.4-0-gf577413>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROC:1]    The new service has started successfully.>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:AGE] Starting reverse tunnel agent pool. servi>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:PRO] Starting Kube proxy on . service/service.>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [DB:SERVIC] Starting Postgres proxy server on 0.0.0.0>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [DB:SERVIC] Starting Database TLS proxy server on 0.0>
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:SER] Starting proxy gRPC server on [::]:3080. >
-Apr 18 18:33:49 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:49Z INFO [PROXY:SER] Starting TLS ALPN SNI proxy server on [::>
-Apr 18 18:33:51 ip-172-30-173-50 teleport[442]: 2022-04-18T18:33:51Z WARN [PROXY:1]   Restart watch on error: empty proxy list.>
+$ teleport install systemd
+```
+
+If the output describes a good configuration for your environment, enter the command once again using the `-o` flag to
+write the output to the directory path of the unit file itself.
+The file path `/etc/systemd/system/teleport.service` is recommended in most use cases as the optimal **unit file load path**,
```

Comment:
I played around with the wording of this, but had a hard time finding a way to mention the recommended load path following the command itself that didn't feel as though it could be confusing to readers. I generally try to describe commands fully before guiding readers to enter them both so that readers can always intuit when the guide is moving on to the next step, and to avoid situations where readers may enter a command that changes their configuration before having an understanding of what the command will be doing. The risk of using the wrong load path in this guide is pretty low, but I do still feel it's worth mentioning for fringe cases